README.md

```diff
@@ -134,9 +134,10 @@ Important options are
   Compiling with maximum rank 15 can be resource intensive and requires at least 16 GB of memory to allow parallel compilation or 4 GB memory for sequential compilation.
 - `-DBUILD_SHARED_LIBS` set to `on` in case you want link your application dynamically against the standard library (default: `off`).
 
-For example, to configure a build using the Ninja backend and generating procedures up to rank 7, which is installed to your home directory use
+For example, to configure a build using the Ninja backend while specifying compiler flags `FFLAGS`, generating procedures up to rank 7, and installing to your home directory, use
 
 ```sh
+export FFLAGS="-O3 -flto"
```

Comment:
```suggestion
export FFLAGS="-O3"
```
I propose to remove "-lto" such that the proposed `FFLAGS` is (almost?) independant of the compiler

Reply:
Thanks, agree that is a good trade-off.